src/flow-control/kill-others.spec.ts

```diff
@@ -23,6 +23,7 @@ const createWithConditions = (conditions: ProcessCloseCondition[]) =>
     new KillOthers({
         logger,
         conditions,
+        killSignal: undefined,
```

Comment:
We need to extend the test file to cover the arguments passed to the `command.kill()`, both when `undefined` and the custom one

Reply:
Fixed.